label_studio_converter/converter.py

```diff
@@ -685,6 +685,11 @@ def convert_to_yolo(self, input_data, output_dir, output_image_dir=None, output_
                     if "rectanglelabels" in label or 'labels' in label:
                         x, y, w, h = self.rotated_rectangle(label)
                         annotations.append([category_id, (x + w / 2) / 100, (y + h / 2) / 100, w / 100, h / 100])
+                    elif "polygonlabels" in label:
+                        points_abs = [(x / 100, y / 100) for x, y in label["points"]]
+                        x, y = zip(*points_abs)
```

Comment:
why do you need this line? x and y are never used below

Reply:
Yeah, that's copy-paste from coco code, will remove that